src/node_url.cc

```diff
@@ -1306,6 +1308,7 @@ namespace url {
           args[3]->IsObject());
     CHECK(args[4]->IsFunction());
     Utf8Value input(env->isolate(), args[0]);
+    CHECK(!input.IsInvalidated());
```

Comment:
I'll return in this and some other cases, but there are instances where `Utf8Value` is used where either returning is inconvenient (because it is an ordinary C++ method, not a V8 binding one) or technically impossible (there is an `IsString()` guard before attempting to read it through `Utf8Value`).